files/scripts/sonic-dpu-mgmt-traffic.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#Script to control the DPU management traffic forwarding through the SmartSwitch
+
+command_name=$0
+
+usage(){
+    echo "Syntax: $command_name -e|--enable -d|--disable"
+    echo "Arguments:"
+    echo "-e			Enable dpu management traffic forwarding"
+    echo "-d			Disable dpu management traffic forwarding"
+}
+
+add_rem_valid_iptable(){
+    local op=$1
+    local table=$2
+    local chain=$3
+    shift 3
+    local rule="$@"
+    iptables -t $table -C $chain $rule &>/dev/null
+    local exit_status=$?
+    local exec_cond=0
+    if [ "$op" = "enable" ]; then
+        exec_command="iptables -t $table -A $chain $rule"
+        [ "$exit_status" -eq 0 ] || exec_cond=1 # Execute if rule is currently not present
+    else
+        exec_command="iptables -t $table -D $chain $rule"
+        [ "$exit_status" -ne 0 ] || exec_cond=1 # Execute if rule is currently present
+    fi
+    if [ "$exec_cond" -eq 1 ]; then
+        eval "$exec_command"
+    else
+        echo "$exec_command not requried, will not be executed"
+    fi
+}
+
+control_forwarding(){
+    local op=$1
+    local value=0
+    if [ "$op" = "enable" ]; then
+        value=1
+    fi
+    echo $value > /proc/sys/net/ipv4/ip_forward
```

Comment:
should we check for 'sudo' permission or 'root' user before running all these commands?

Reply:
Done, Added check to see if script is executed in `sudo` mode or not